Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.8-alpine
```

Comment:
@konklone: the question comes down to whether we stick with the 18F best practice that is written to give better parity to cloud.gov, or if we break from that since this repo is popular amongst folks who aren't working within cloud.gov. The argument for breaking away is that we get a much slimmer image as Paolo demonstrated 👆